IHP/IDE/SchemaDesigner/Controller/Validation.hs

```diff
@@ -0,0 +1,22 @@
+module IHP.IDE.SchemaDesigner.Controller.Validation where
+
+import IHP.ControllerPrelude
+import IHP.IDE.SchemaDesigner.View.Layout (isIllegalKeyword)
```

Comment:
Wouldn't it make sense to move the `isIllegalKeyword` from the `Layout.hs` into this file here as well? :) Then the `Layout.hs` could just import it from here.